beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1319,8 +1320,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         block: SignedBeaconBlock<T::EthSpec>,
     ) -> Result<GossipVerifiedBlock<T>, BlockError<T::EthSpec>> {
         let slot = block.message.slot;
-        let graffiti_string = String::from_utf8(block.message.body.graffiti[..].to_vec())
-            .unwrap_or_else(|_| format!("{:?}", &block.message.body.graffiti[..]));
+        let re = Regex::new(r"\p{C}").unwrap();
```

Comment:
To work around the Clippy issue we can write `"\\p{C}"` here instead. I opened https://github.com/rust-lang/rust-clippy/issues/6005 to track the false positive.

The `unwrap` would maybe be better as an `.expect("regex is valid")`